tests/Windows.Win32.Tests/assets/suggestedRemappingsAllowList.rsp

```diff
@@ -258,4 +258,5 @@ Gdiplus::BrushType=GpBrushType
 Gdiplus::MatrixOrder=GpMatrixOrder
 Gdiplus::FlushIntention=GpFlushIntention
 Gdiplus::PathData=GpPathData
-_KNONVOLATILE_CONTEXT_POINTERS_ARM64=KNONVOLATILE_CONTEXT_POINTERS_ARM64
\ No newline at end of file
+_KNONVOLATILE_CONTEXT_POINTERS_ARM64=KNONVOLATILE_CONTEXT_POINTERS_ARM64
+_DXDIAG_INIT_PARAMS=DXDIAG_INIT_PARAMS
```

Comment:
This should be remapped in your partition, not ignored (allowed). Add:

```
--remap
_DXDIAG_INIT_PARAMS=DXDIAG_INIT_PARAMS
```

to `settings.rsp` in your partition.

Reply:
Yeah, I fall into this trap myself too. The warning makes it very easy to do the wrong thing.

Your previous remap is fine as-is, but to answer your question: yes probably. I'm not sure what the status of metadata is at the moment, so I would hold off on adding another PR.